src/main/java/org/fundacion/jala/converter/core/parameter/Parameter.java

```diff
@@ -8,14 +8,11 @@
  *
```

Comment:
In this file in line 4 should be removed the "* " before This

Reply:
Done.